Client/src/Unified/Services/Submitter/BatchUntilInactiveBlock.cs

```diff
@@ -0,0 +1,249 @@
+// This file is part of the ArmoniK project
+//
+// Copyright (C) ANEO, 2021-$CURRENT_YEAR$. All rights reserved.
```

Comment:
YEAR

Reply:
It is written like that in the .editorconfig so it will appear in every new file